src/components/data-management/ProjectDetails.jsx

```diff
@@ -481,10 +481,11 @@ const ProjectDetails = ({ width, height }) => {
       // Check if all files for a given tech has been uploaded
       const fileNamesArray = Array.from(sample.fileNames);
 
-      if (!_.isEqual(
```

Comment:
`_.isEqual` checks for order. `fileNamesArray` may not be uploaded in the same order as `fileUploadSpecifications[sample.type].requiredFiles` hence it fails.